source/core/parameters_lagrangian.cc

```diff
@@ -605,21 +606,33 @@ namespace Parameters
                         "1",
                         Patterns::Integer(),
                         "Output frequency");
-      prm.enter_subsection("center of mass coordinate");
-      prm.declare_entry("x",
-                        "0",
-                        Patterns::Double(),
-                        "X coordinate of center of mass");
-      prm.declare_entry("y",
-                        "0",
-                        Patterns::Double(),
-                        "Y coordinate of center of mass");
-      prm.declare_entry("z",
+      prm.declare_entry("center of mass coordinate",
+                        "0,0,0",
+                        Patterns::List(Patterns::Double(),3,3),
+                        "Coordinate of center of mass");
+      prm.declare_entry("boundary inertia",
+                        "0,0,0",
+                        Patterns::List(Patterns::Double(),3,3),
+                        "Boundary inertia around x,y,z axis");
+      prm.declare_entry("boundary mass",
                         "0",
                         Patterns::Double(),
-                        "Z coordinate of center of mass");
-      prm.leave_subsection();
+                        "Boundary mass in kg");
```

Comment:
What is boundary mass? Is it the mass of the solid body that is moving?

Reply:
Yes